plugins/Ecommerce/Columns/Order.php

```diff
@@ -19,13 +20,16 @@ class Order extends BaseConversion
     protected $columnName = 'idorder';
     protected $type = self::TYPE_NUMBER;
     protected $category = 'Goals_Ecommerce';
-    protected $nameSingular = 'Ecommerce_Order';
+    protected $nameSingular = 'Ecommerce_OrderId';
     protected $namePlural = 'Ecommerce_Orders';
     protected $metricId = 'orders';
+    protected $segmentName = 'orderId';
 
     public function configureMetrics(MetricsList $metricsList, DimensionMetricFactory $dimensionMetricFactory)
     {
         $metric = $dimensionMetricFactory->createMetric(ArchivedMetric::AGGREGATION_UNIQUE);
+        $metric->setName("ecommerce_order_id");
```

Comment:
@katebutler do you remember if the name was needed? Changing the name of existing metrics may break existing custom reports that have the old metricId assigned.

Reply:
@tsteur Turns out it's not needed, have removed it